Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -318,29 +319,62 @@ export class dotNetExe {
     }
 
     private getProjectFiles(): string[] {
-        var projectPattern = this.projects;
-        var searchWebProjects = this.isPublishCommand() && this.publishWebProjects;
+        let projectPattern = this.projects;
+        const searchWebProjects: boolean = this.isPublishCommand() && this.publishWebProjects;
         if (searchWebProjects) {
-            projectPattern = ["**/*.csproj", "**/*.vbproj", "**/*.fsproj"];
+            projectPattern = ['**/*.csproj', '**/*.vbproj', '**/*.fsproj'];
         }
 
-        var projectFiles = utility.getProjectFiles(projectPattern);
+        const projectFiles = utility.getProjectFiles(projectPattern);
+        let resolvedProjectFiles: string[] = [];
 
         if (searchWebProjects) {
-            projectFiles = projectFiles.filter(function (file, index, files): boolean {
-                var directory = path.dirname(file);
-                return tl.exist(path.join(directory, "web.config"))
-                    || tl.exist(path.join(directory, "wwwroot"));
+            resolvedProjectFiles = projectFiles.filter(function (file, index, files): boolean {
+                const directory = path.dirname(file);
+                return tl.exist(path.join(directory, 'web.config'))
+                    || tl.exist(path.join(directory, 'wwwroot'));
             });
 
-            if (!projectFiles.length) {
-                tl.error(tl.loc("noWebProjctFound"));
+            if (!resolvedProjectFiles.length) {
+                projectFiles.forEach((projectFile) => {
+                    if (isWebSdkUsed(projectFile)) {
+                        resolvedProjectFiles = projectFile;
+                    }
+                });
+                tl.error(tl.loc('noWebProjctFound'));
             }
         }
 
         return projectFiles;
     }
 
+    private isWebSdkUsed(projectfile: string) {
+        var fileBuffer: Buffer = fs.readFileSync(projectfile);
+        var fileEncodeType = fileEncoding.detectFileEncoding(projectfile, fileBuffer);
+        var webConfigContent: string = fileBuffer.toString(fileEncodeType[0]);
+        if(fileEncodeType[1]) {
+            webConfigContent = webConfigContent.slice(1);
+        }
+        var xmlDocument;
+        var paramFileReplacableValues = {};
+        try {
+            var ltxDomUtiltiyInstance = new ltxdomutility.LtxDomUtility(webConfigContent);
+            xmlDocument = ltxDomUtiltiyInstance.getXmlDom();
+            for(var xmlChildNode of xmlDocument.children) {
+                if(!varUtility.isObject(xmlChildNode)) {
+                    continue;
+                }
+                if(xmlChildNode.name == "project" && !!xmlChildNode.attrs['sdk']) {
+                    if (xmlChildNode.attrs['sdk'] == ""
```

Comment:
is this incomplete code?

Reply:
@sachinma 
The bug is being taken care by Nitin now, i was only transferring the work done so far to him.
Nitin will take it further and raise proper PR.